common/v2/services/Contracts/types.ts

```diff
@@ -0,0 +1,10 @@
+export interface Contract {
+  name: string;
+  networkId: string;
+  address: string;
+  abi: string;
```

Comment:
If the ABI is always JSON, we should store it as a specific type, or `unknown` if the structure is unknown or differs between contracts

Reply:
I've make a task to handle this in a different pr.